libtextsecure/test/crypto_test.js

```diff
@@ -0,0 +1,54 @@
+describe('encrypting and decrypting profile data', function() {
```

Comment:
Could we perhaps name this file `profile_crypto_test.js` if it's specific to profiles?